site/src/docs/getting-started/tutorials/migrate-to-4.0.0.mdx

```diff
@@ -195,75 +195,95 @@ There are updates to lineheights to improve readability especially on Headings.
 #### Black-60 changed to more accesible shade
 Design token black-60 is changed from #666666 to #595959 to improve contrast with other colours. With only bus-dark and black-90 the contrast gets worse, all other colours the contrast improves. If needed, verify the used colours.
 
+---
+
 ## Design kit
```

Comment:
I would add 'Figma' to the title since all of these changes are guiding changes there

```suggestion
## Figma Design Kit
```